src/utils/string.ts

```diff
@@ -1,9 +1,9 @@
-export const capitalize = (str) => {
+export const capitalize = (str: string) => {
   const lower = str.toLowerCase();
   return str.charAt(0).toUpperCase() + lower.slice(1);
 };
 
-export const leftPad = (number, digits) => {
+export const leftPad = (number: number, digits: number): string => {
```

Comment:
idem qui, ma in questo caso ti consiglio di usare la ormai versione nativa di leftPad!

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart